documentation/translations/coordinating.rst

```diff
@@ -3,79 +3,184 @@ Coordinating
 
-- Create the GitHub repo (any account) with the correct hierarchy by using one
-  of the bootstrappers or Transifex.
-- Gather people to help you translate. You can't do it alone.
-- You can use any tool to translate, as long as you can synchronize with Git.
-  Some use Transifex, and some use only GitHub. You can choose another
-  way if you like; it's up to you.
-- Update :doc:`this page <translating>` to reflect your work and progress, either via a
-  PR or by asking on the `translation mailing list <translation_ml_>`_.
-- When ``bugs``, ``tutorial``, and ``library/functions`` are 100%
-  completed, ask on the `translation mailing list <translation_ml_>`_ for
-  your language to be added in the language switcher on docs.python.org.
+Transifex
+~~~~~~~~~
 
+You can also start your translation using `Transifex <https://explore.transifex.com/python-doc/python-newest/>`_.
+This will automate updating translations, and will allow you to translate via
+the web interface.
 
-How to get help
-===============
+This is best done with a workflow that periodically checks for translations,
+a sample one with instructions can be found `here <https://python-docs-transifex-automation.readthedocs.io/workflows.html>`__.
+An in depth guide for manually doing this can also be found `here <https://python-docs-transifex-automation.readthedocs.io/commands.html>`__.
 
-Discussions about translations occur on the Python Docs Discord
-`#translations channel <https://discord.gg/h3qDwgyzga>`_, `translation
-mailing list <translation_ml_>`_, and the
-`translations category <https://discuss.python.org/c/documentation/translations/>`_
-of the Python Discourse.
+To be added as the coordinator(s) on Transifex for your language, open an issue
+in the `tracker <https://github.com/python-docs-translations/transifex-automations/issues>`__.
+
+
+Others
+~~~~~~
+
+You can choose another way if you like; it’s up to you. Getting it right from
+the start will prevent complications in the future when your translation
+is built. If complications do occur more information about them can be found
+in the FAQ section on this page.
+
+
+Glossary
+--------
+
+Each translation team should have a way to store translations of terms to ensure
+consistency. This is often done with a glossary. More information about the use
+of glossaries can be found in the :ref:`translation-style-guide`.
+
+
+Moving the repo to the python org
+---------------------------------
+
+This will allow you to plug your translation into docsbuild-scripts_, and it
+will be found at ``docs.python.org/LANG/``, but not in the switcher.
+
+This will be done XXX
+
+.. XXX When ...? Discussion needed...
+.. My idea: Time based, e.g. 2 months of activity, showing that they aren't going anywhere
+
+Adding to the language switcher
+-------------------------------
+
+.. XXX https://github.com/python/devguide/issues/1586
+
+Once the following resources have been fully translated on the XXX branch:
```

Comment:
I guess these XXX requires closing that issue before merging this? Alternatively, you could use a text that doesn't mention which branch, and leave the branch requirement for another PR.